src/main/webapp/i18n/de/exercise.json

```diff
@@ -69,6 +69,8 @@
             "startError": "<strong>Uh oh! Etwas ist schief gelaufen... Bitte versuch es in wenigen Minuten noch einmal die Aufgabe zu starten.</strong>",
             "studentName": "Name",
             "studentId": "Login",
+            "teamName": "Team Name",
+            "teamShortName": "Kurzname",
```

Comment:
Is it important to state that it's the short name of the team?

Reply:
You are right that this is a little inconsistent when looking at the translation of `teamName` vs `teamShortName`. The reason why I left out the `Team` in the translation of `teamShortName` is that it gets used right in the column header next to `Team Name`, so that it is obvious from context that the short name refers to the team and I wanted to save horizontal space since the column otherwise doesn't necessarily need that much width.

But it does feel like a bad practice to "cheat" with the translation for a specific use case. What I'll do: I'll rename this translation to `Team Kurzname` and add another translation pair just for `shortName` (if it doesn't already exist elsewhere). I think that's a good solution.

Reply:
Hm, on second thought that doesn't make too much sense either since the two student-related translations above also don't follow that kind of schema. I'd now probably tend to keep it as is where `teamName` is kind of the exception in that it contains the entity name in the translation value but I think that makes sense to differentiate it from the value of `studentName` (although in context it will probablt be obvious that those names are not student names but team names; the exception is when the table does not contain any rows).